docs/execution-providers/TensorRTRTX-ExecutionProvider.md

```diff
@@ -209,20 +235,20 @@ TensorRT RTX configurations can be set by execution provider options. It's usefu
   * Note that multiple TensorRT RTX profiles can be enabled by passing multiple shapes for the same input tensor.
   * Check TensorRT doc [optimization profiles](https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#opt_profiles) for more details.
 
-## NV TensorRT RTX EP Caches
-There are two major TRT RTX EP caches:
-* Embedded engine model / EPContext model
-* Internal TensorRT RTX cache
+## TensorRT RTX EP Caches
+There are two major types of caches in TensorRT RTX EP :
+* EP context model
+* Internal TensorRT RTX cache (TODO)
 
-The internal TensorRT RTX cache is automatically managed by the EP. The user only needs to manage EPContext caching. 
+The internal TensorRT RTX cache is automatically managed by the EP. The user only needs to manage EP context caching. 
 **Caching is important to help reduce session creation time drastically.**
 
-TensorRT RTX separates compilation into an ahead of time (AOT) compiled engine and a just in time (JIT) compilation. The AOT compilation can be stored as EPcontext model, this model will be compatible across multiple GPU generations.
-Upon loading such an EPcontext model TensorRT RTX will just in time compile the engine to fit to the used GPU. This JIT process is accelerated by TensorRT RTX's internal cache.
+TensorRT RTX separates compilation into an ahead of time (AOT) compiled engine and a just in time (JIT) compilation. The AOT compilation can be stored as EP context model, this model will be compatible across multiple GPU generations.
+Upon loading such an EP context model, TensorRT RTX will just in time compile the engine to fit to the used GPU. This JIT process is accelerated by TensorRT RTX's internal cache.
```

Comment:
Currently there is no internal cache in use. This is something we still have to merge with runtime caches.